configs/celery/phsa_celery_nmap.conf

```diff
@@ -0,0 +1,22 @@
+# place or link this file under /etc/conf.d/ or whatever path is used for
+# the EnvironmentFile value in the .service file
+
+# ok, this is a hack and it needs to go away as soon as we do the dedicated user stuff
+C_FORCE_ROOT=1
+
+CELERY_APP="p_soc_auto"
+CELERYD_NODES="p_soc_auto_celery_nmap"
+
+CELERYD="multi"
+
+# celery worker options
+# autoscale up to 10 processes, never run less than 3 processes
+# concurrency, 8 threads per process
+# soft-time-limit, throw catchable exception, kill tasks longer than 10mins
+# send the events, maybe we will disable this
+CELERYD_OPTS="--queues=nmap --autoscale=10,3 --soft-time-limit=300 --events --max-tasks-per-child=8"
+
+CELERY_BIN="/home/steodore/.envs/carmina/bin/celery"
```

Comment:
I understand we will point celery_bin to a /opt/bin.... in future
but for a time being we should use
something similar like following to avoid less editing when checking out our codes
CELERY_BIN="/virtualenvs/def/bin/celery"

Reply:
since the story for that is already on the board, can we consider this reviewed and approved?